### ISSUE TITLE
How is the Github Windows client made?

### ISSUE THREAD

**Issue by yasoob** [holman/feedback#633]
Sorry for that direct question. Let me rephrase it. I would love it if you can tell me which GUI toolkit do you guys use. Which language is utilized and how much custom styling is employed. 

Thanks in advance :+1:

**Comment by holman** [comment]
I bet if we ask @jspahrsummers real nicely he could swing in and chat real briefly about some of the rad open source projects they've spun out from it.

**Comment by holman** [comment]
oh christ you said Windows not Mac my bad

@haacked we don't even have to ask nicely we can just demand it and he does anything to get the limelight.

Sorry Justin!

**Comment by jspahrsummers** [comment]
It's okay, I know I don't have what it takes…

![](http://www.hanselman.com/blog/content/binary/WindowsLiveWriter/3b7c9338d193_9C2C/277527851_3.jpg)

**Comment by Haacked** [comment]
Ain't nobody going to look that good!

If you go to the About menu and click "Licenses" you can see a list of a lot of the libraries we use.

![280](https://cloud.githubusercontent.com/assets/19977/5653796/4c8bd93c-9672-11e4-8ba6-32ba21087aba.png)

The application is a WPF application with a lot of custom styling.

**Comment by holman** [comment]
I'm going to use that image everywhere now!

**Comment by Haacked** [comment]
At least at GHfW in there now.

**Comment by Haacked** [comment]
![selfie-0](http://i.imgur.com/VWsmmNx.gif)

**Comment by Haacked** [comment]
Yup. C#

**Comment by yasoob** [comment]
And the last question. How long did it take you guys to go from the concept
to the first working prototype and then the final product.

**Comment by Haacked** [comment]
Hard to say. The first commit was in May 2010, but it was just a README file with no code. But you could argue that's when the concept was created. There wasn't another commit until a year later in June 2011. This was before I was at GitHub. We shipped GitHub 1.0 a year later in May 21, 2012.

**Comment by yasoob** [comment]
oh ok. Thanks :)
